### PR TITLE
cli: allow to calculate contract hash before deployment

### LINKED ISSUE #1613: Add CLI command to calculate deployed contract hash
NEF and sender in --- hash out, simple as that.

### PR DESCRIPTION
**Pull request by fyrchik**
Close #1613 .

### CODE REVIEW

File: cli/smartcontract/smart_contract.go [hunk 1]
```diff
@@ -337,6 +337,21 @@ func NewCommands() []cli.Command {
...
 				},
 			},
+			{
+				Name:   "calc-hash",
+				Usage:  "calculates hash of a contract after deployment",
+				Action: calcHash,
+				Flags: []cli.Flag{
+					cli.StringFlag{
+						Name:  "sender, s",
+						Usage: "sender script hash or address",
```
Comment (AnnaShaleva) [inline cli/smartcontract/smart_contract.go:347]: Maybe add here that script hash should be in LE?

File: cli/smartcontract/smart_contract.go [hunk 2]
```diff
@@ -337,6 +337,21 @@ func NewCommands() []cli.Command {
 					},
 				},
 			},
+			{
+				Name:   "calc-hash",
+				Usage:  "calculates hash of a contract after deployment",
```
Comment (AnnaShaleva) [inline cli/smartcontract/smart_contract.go:342]: `before` deployment?

**Review by AnnaShaleva — CHANGES_REQUESTED** [review]

File: cli/smartcontract/smart_contract.go (hunk 1 above)
Comment (roman-khimov) [reply cli/smartcontract/smart_contract.go:347]: All user-facing things are usually LE.

File: cli/smartcontract/smart_contract.go (hunk 2 above)
Comment (roman-khimov) [reply cli/smartcontract/smart_contract.go:342]: It's calculating what the hash will be after deployment before deployment, yeah.

**Review by AnnaShaleva — APPROVED** [review]

**Pull request merged**